German translations and review existing translations

rebplu opened the pull request.

rebplu committed:
Update calendar.ttl

German Translations Comments to be checked

rebplu committed:
Update catholic-order.ttl

German translations; comment to the different orders still missing

rebplu committed:
Update catholic-organization.ttl

German translations

rebplu committed:
Update concept.ttl

German translations

rebplu committed:
Update creating.ttl

German translations

rebplu committed:
Update device.ttl

some little corrections in German translations

rebplu committed:
Update document.ttl

review German translations ("expression" in German = "Ausdruck"?!)

rebplu committed:
Update event.ttl

German translations (one unclear commented (en: 'staging'))

rebplu committed:
Update human-geography.ttl

Review German translations

rebplu committed:
Update human.ttl

Review German translations

rebplu committed:
Update human.ttl

little Changes to the commit before

rebplu committed:
Update calendar.ttl

2. Review

rebplu committed:
Update agent.ttl

agent's role = agent role
in German: Erschaffer = Ersteller

rebplu committed:
Update creating.ttl

Changes and Questions.

rebplu committed:
Update calendar.ttl

after discussion

rebplu committed:
Update concept.ttl

change after discussion

rebplu committed:
Update creating.ttl

Changes after discussion

rebplu committed:
Update human.ttl

hanscools commented on Nie-ontologies/Generic-ontologies/calendar.ttl:
```diff
@@ -260,7 +260,7 @@ calendar:julianListToJDN
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/calendar#>;
 	a owl:DatatypeProperty;
-	rdfs:label "Julian set list to date"@en, "julianische Reihenliste zu Datum"@de;#
+	rdfs:label "set Julian date list to date"@en, "gesetzte julianische Datumliste zu Datum"@de;
```
OK

hanscools commented on Nie-ontologies/Generic-ontologies/creating.ttl:
```diff
@@ -45,21 +45,21 @@ creating:AgentCreator
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/creating#>;
 	a rdfs:Class;
-	rdfs:label "creator role of person"@en, "Erstellerrolle eines Akteur"@de;
+	rdfs:label "creator role of person"@en, "Erstellerrolle eines Akteurs"@de;
```
"Erstellerrolle eines Akteurs" -> "Erstellerrolle einer Person"

hanscools commented on Nie-ontologies/Generic-ontologies/creating.ttl:
```diff
@@ -45,21 +45,21 @@ creating:AgentCreator
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/creating#>;
 	a rdfs:Class;
-	rdfs:label "person-creating"@en, "Erstellen"@de; #why not just creating (similar to the next (creation))
+	rdfs:label "creating"@en, "Erstellung"@de; #why not just creating (similar to the next (creation))
```
Remove comment.
reb: last commit in creating.ttl did not work. New commit 9eeda18 with all discussed changes.

rebplu committed:
Update creating.ttl

harmonize 'person creation', 'person's creation', 'person-creation' or just 'creation' to always 'creation' (after discussion, that it is clear from the ontology-title a.s.o, that it is a person's creation)
Remove comments

hanscools commented on Nie-ontologies/Generic-ontologies/human.ttl:
```diff
@@ -184,28 +184,28 @@ human:AgentUser
 	a rdfs:Class;
 #	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human#>;
-	rdfs:label "agent user"@en, "benutzender Akteur"@de;
+	rdfs:label "agent role of user"@en, "Akteurrolle eines Benutzers"@de;
```
"agent role of user" -> "user role of agent"; idem for the other roles
reb: ups sorry for that...

rebplu committed:
Update human.ttl

rebplu commented:
Shall we try to merge? I will then go over the changed files again briefly. But rather in a new branch. I have no overview anymore after so many commits...:-)

Commit:
Merge branch 'master' into germanTranslations_and_validations_rebplu
